lib/fastlane_core/simulator.rb

```diff
@@ -6,32 +6,23 @@ class << self
       def all
         return @devices if @devices
         Helper.log.info "Fetching available devices" if $verbose
-
-        # we do it using open since ` just randomly hangs with instruments -s
-        output = ''
-        Open3.popen3('instruments -s') do |stdin, stdout, stderr, wait_thr|
-          output = stdout.read
-        end
-
-        unless output.include?("Known Devices")
-          Helper.log.error "Instruments CLI broken, run `instruments -s` and make sure it works".red
-          Helper.log.error "The easiest way to fix this is to restart your Mac".red
-          raise "Instruments CLI not working.".red
-        end
-
-        output = output.split("Known Devices:").last.split("Known Templates:").first
+        
         @devices = []
-        output.split("\n").each do |current|
-          m = current.match(/(.*) \((.*)\) \[(.*)\]/)
-          next unless m
-          name = m[1].to_s.strip
-          ios = m[2].to_s.strip
-          udid = m[3].to_s.strip
-          next unless udid.include?("-") # as we want to ignore the real devices
-
-          @devices << Device.new(name: name, ios_version: ios, udid: udid)
+        (os_type, os_version) = ['unknown', 'unknown']
+        `xcrun simctl list devices`.split(/\n/).each do |line|
+          next if line.match /^== /
+          if line.match /^-- /
+            (os_type, os_version) = line.gsub(/-- (.*) --/, '\1').split
+          else
+            # iPad 2 (0EDE6AFC-3767-425A-9658-AAA30A60F212) (Shutdown)
+            # iPad Air 2 (4F3B8059-03FD-4D72-99C0-6E9BBEE2A9CE) (Shutdown) (unavailable, device type profile not found)
+            match = line.match /\s+([^\(]+) \(([-0-9A-F]+)\) \((?:[^\(]+)\)(.*unavailable.*)?/
```

Comment:
Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.